nltk/tokenize/stanford_segmenter.py

```diff
@@ -46,10 +46,9 @@ class StanfordSegmenter(TokenizerI):
     """
 
     _JAR = 'stanford-segmenter.jar'
-    _SLF4J = 'slf4j-api.jar'
 
     def __init__(self,
-                 path_to_jar=None, path_to_slf4j=None,
```

Comment:
keep `path_to_slf4j`, but make it default to `'slf4j-api.jar'`. Also move it to the new line.